tsdb/engine/tsm1/file_store.go

```diff
@@ -366,9 +387,7 @@ func (f *FileStore) Open() error {
 		go func(idx int, file *os.File) {
 			start := time.Now()
 			df, err := NewTSMReader(file)
-			if f.traceLogging {
-				f.Logger.Printf("%s (#%d) opened in %v", file.Name(), idx, time.Now().Sub(start))
-			}
+			f.traceLogger.Printf("%s (#%d) opened in %v", file.Name(), idx, time.Now().Sub(start))
```

Comment:
This needs to be regular logger.  Otherwise, an operator has no idea if startup is progressing.